apps/storybook/stories/Namekit/DisplayedPrice.stories.tsx

```diff
@@ -0,0 +1,299 @@
+import { Currency } from "@namehash/ens-utils";
+import {
+  AltPriceDisplayFormat,
+  DisplayedPrice,
+  PriceDisplayPosition,
+  PriceDisplaySize,
+  PriceSymbolPosition,
+  PriceSymbology,
+} from "@namehash/namekit-react";
+import type { Meta, StoryObj } from "@storybook/react";
+
+const meta: Meta<typeof DisplayedPrice> = {
+  component: DisplayedPrice,
+  title: "Namekit/DisplayedPrice",
+};
+
```

Comment:
Is my memory correct that we also have defined the concept of a maximum displayed price in each currency? It would be great to show those examples in storybook for each currency.

Additionally, depending on the currency, there can be cases where the price is > 0 but it displays as as some minimum displayed price. It would also be great to add those examples here.

Reply:
Added for both ETH and USD prices!